starknet_py/tests/e2e/conftest.py

```diff
@@ -194,6 +197,20 @@ def compiled_proxy(request) -> str:
     return (directory_with_contracts / request.param).read_text("utf-8")
 
 
+@pytest.fixture(name="typed_data")
```

Comment:
This could be a parametrized fixture but this can be addressed in separate PR once #356 is merged.